src/video/H264Encoder.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "encoder.hpp"
+
+#include <string>
+#include <vector>
+
+#include <opencv2/core/core.hpp>
+
+namespace video {
+/**
+ * This class is a wrapper for the h264encoder library
```

Comment:
This isn't quite correct anymore. Plus, we don't care what it's a wrapper for. Explain what the class does, how it does it is abstracted away from the client.